### ISSUE TITLE
[hue] Add Hue Sync Box

### ISSUE THREAD

**Issue by WhiteSockedDancer** [openhab/openhab-addons#10218]
Hey, is there any way to integrate a Hue Sync Box into this Binding?
It would be great if we could control Input-Sources, Sync, Mode and so on over OH :)

https://www.philips-hue.com/de-de/p/hue-play-hdmi-sync-box/8718699704803#features

Greetings

**Comment by pgfeller** [comment]
I'll have a look into the work done by [Homey-GER](https://github.com/Homey-GER) in https://github.com/Homey-GER/openhab-addons to see if I can create an official binding based on his code. According to the Philips Developer Documentation the HDMI Bridge is discoverable by mDNS [Hue HDMI Sync Box API](https://developers.meethue.com/develop/hue-entertainment/hue-hdmi-sync-box-api/) - which would nicely fit with the new binding proposal mechanism of openHAB 4. Also the API seems well documented.

However - assuming I'm able to create a working binding, I'll not have the resources to be an official maintainer of a binding ...

I'll use this issue to post progress (if there's any 🙂) and to get feedback etc. If this is the wrong place to do this (e.g. the discussion feature, or the forum shall be used for this), please let me know.

**Comment by pgfeller** [comment]
- [x] ☠️ Binding skeleton created for org.openhab.binding.huesync
- [x] 🔎 skeleton mDNS discovery implemented
- [x] 🔎 mDNS device discovery - use API to get device information
- [x] 🔐 SSL Handshake & 🔎 Discovery
- [ ] 🤝 Device registration
- [ ] ...

**Comment by lolodomo** [comment]
Isn't it possible to simply enhance the existing hue binding ?

**Comment by andrewfg** [comment]
The Hue Sync box is a completely different piece of hardware than the Hue Bridge. It intercepts an HDMI stream, and sends entertainment based commands to a Hue Bridge to control the lighting. There are **two different boxes on the LAN** each with their own IP addresses and own APIs. The Hue Sync box has a W-LAN interface on one IP address that supports one API for setting the entertainment functions. The Hue Bridge has its own LAN interface on another IP address that supports two APIs for direct lighting control. The Hue Bridge does have a couple of API endpoints for ['entertainment'](https://developers.meethue.com/develop/hue-api-v2/api-reference/) however these are NOT for commanding the Hue Sync Box functionality but rather (potentially) for setting up how the Hue Bridge responds to commands received from the Hue Sync Box. 

TLDR these are two completely different: physical hardware / IP addresses /  software interfaces => there is NO point in trying to incorporate the Hue Sync Box functionality in the Hue Bridge binding. It requires a complete new binding!
